CMakeLists.txt

```
@@ -35,6 +35,8 @@ endif()
 option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
 option(SOLC_STATIC_STDLIBS "Link solc against static versions of libgcc and libstdc++ on supported platforms" OFF)
 option(STRICT_Z3_VERSION "Use the latest version of Z3" ON)
+option(SOL_FORCE_RELEASE "Force the creation of a Release Version" OFF)
```

```suggestion
option(SOL_FORCE_RELEASE "Build a release binary" OFF)
```

Maybe we should really call this `SOL_RELEASE_VERSION`? Force sounds a bit as if you were doing something you should not be doing but this is now **the** way to build a release binary.

---

Sure, will change it in the next iteration